databricks/koalas/missing/series.py

```diff
@@ -154,12 +147,19 @@ class _MissingPandasLikeSeries(object):
     select = unsupported_function('select', deprecated=True)
     set_value = unsupported_function('set_value', deprecated=True)
     valid = unsupported_function('valid', deprecated=True)
+    get_values = unsupported_function('get_values', deprecated=True)
+    to_dense = unsupported_function('to_dense', deprecated=True)
+    to_sparse = unsupported_function('to_sparse', deprecated=True)
+    to_msgpack = unsupported_function('to_msgpack', deprecated=True)
 
-    # Functions and properties we won't support.
+    # Properties we won't support.
     values = common.values(unsupported_property)
+    array = common.array(unsupported_property)
+    real = unsupported_property(
+        'real',
+        reason="If you want to collect your data as an NumPy array, use 'to_numpy()' instead.")
+
+    # Functions we won't support.
     memory_usage = common.memory_usage(unsupported_function)
-    # Functions and properties we won't support.
-    to_pickle = unsupported_function(
-        'to_pickle',
-        reason="For storage, we encourage you to use Delta or Parquet, instead of Python pickle "
-               "format.")
+    to_pickle = common.to_pickle(unsupported_function)
+    to_xarray = common.to_xarray(unsupported_function)
```

Comment:
Seems like `to_list` is already implemented in Series. Let me make a separate PRs to handle:

1. "hidden" deprecated APIs in Pandas (like `is_copy`)
2. Audit existing APIs like `to_list`.